mmv1/third_party/terraform/services/filestore/resource_filestore_instance_test.go

```diff
@@ -190,3 +214,27 @@ resource "google_filestore_instance" "instance" {
 }
 `, name)
 }
+
+func testAccFileInstanceTags(name string, tags map[string]string) string {
+	return fmt.Sprintf(`
+resource "google_filestore_instance" "instance" {
+  name = "tf-instance-%s"
+  zone = "us-central1-b"
+  tier = "BASIC_HDD"
+
+  file_shares {
+    capacity_gb = 1024
+    name        = "share1"
+  }
+
+  networks {
+    network           = "default"
+    modes             = ["MODE_IPV4"]
+    reserved_ip_range = "172.19.31.0/29"
+  }
+tags = {
```

Comment:
Do not hardcode tags here please. Use the parameter to build the tags field. Similar to here: https://github.com/GoogleCloudPlatform/magic-modules/pull/11440/files